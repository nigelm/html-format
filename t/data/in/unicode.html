<html>
<body>
<p>French: hétérogénéité, ça a beaucoup d'accents.</p>

<p>Polish: welcome to sunny Łódź.</p>

<p>Here's how you say "tax return" in Thai:
คำร้องขอคืนเงินภาษีอากร</p>
	
<p>This Mongolian person's twitter username character is not a
soft-hyphen:<br/>
ᠭ-ᠪᠢᠰᠢᠷᠡᠯ</p>

<p>Let us all worship at the shrine of Emoji: how else could you throw
a pile of poo at a snowman?<br/>
💩→☃</p>

<p>This sentence is full of non-breaking spaces but it'll be broken anyway.</p>

<p>I put a hyphen in llanfairpwllgwyngyllgogerychwyrndrob­wllllantysiliogogogoch
and it ended up as a space.</p>


</body>
</html>