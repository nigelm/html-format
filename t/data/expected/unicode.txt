     French: h√©t√©rog√©n√©it√©, √ßa a beaucoup
     d'accents.

     Polish: welcome to sunny ≈Å√≥d≈∫.

     Here's how you say "tax return" in Thai:
     ‡∏Ñ‡∏≥‡∏£‡πâ‡∏≠‡∏á‡∏Ç‡∏≠‡∏Ñ‡∏∑‡∏ô‡πÄ‡∏á‡∏¥‡∏ô‡∏†‡∏≤‡∏©‡∏µ‡∏≠‡∏≤‡∏Å‡∏£

     This Mongolian person's twitter username
     character is not a soft-hyphen:
     ·†≠-·†™·†¢·†∞·†¢·†∑·†°·†Ø

     Let us all worship at the shrine of Emoji: how
     else could you throw a pile of poo at a
     snowman?
     Ì†ΩÌ≤© ‚Üí ‚òÉ

     This sentence is full of non-breaking spaces
     but it'll be broken anyway.

     I put a hyphen in
     llanfairpwllgwyngyllgogerychwyrndrob
     wllllantysiliogogogoch and it ended up as a
     space.

